pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -326,6 +329,9 @@ func (r *Builder) mapCPU(vm *model.VM, object *cnv.VirtualMachineSpec) {
 		vm.CoresPerSocket = 1
 	}
 	object.Template.Spec.Domain.Machine = &cnv.Machine{Type: "q35"}
```

Comment:
that's an indication that this part does not belong here
CC @bkhizgiy 
@liranr23 could you please clean it up and make the changes here similar to those in the builder of openstack?

Reply:
i might have made it quite the opposite, please revisit